backend/app/api/routes/model_management.py

```
@@ -0,0 +1,263 @@
+from app.core.config import settings
+from app.models import User
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+class ModelConfig(BaseModel):
+    """模型配置结构"""
+    model_name: str
+    litellm_params: Dict[str, Any]
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)